akirachix/Taishi-Informational
# added footer section

# Engineers Checklist

This PR adds a footer component to the application, designed to provide essential information such as contact details, resources, and legal links, while maintaining a visually appealing layout that adapts across different screen sizes.

The addition of this footer component is necessary to improve the user experience by providing easy access to important information at the bottom of every page. The footer is designed to be responsive and consistent with the overall branding of the application.

* What is this?
This PR creates a new footer component for the application.

* Why this?
To ensure that all pages have a consistent footer with important information like contact details, resources, and legal links.

* How are you doing this? 
The footer is created using a combination of Flexbox utilities for layout management (flex, justify-between, etc.).
The design includes sections for "Themis AI," "Contact Us," "Resources," and "Legal," each spaced equally to provide a clean and professional appearance.
Implemented media queries to ensure it looks good on all screen sizes.




## How can it be tested?
View the Footer on All Pages: Open any page of the application to verify that the footer is present and correctly aligned.
Test Responsiveness: Resize the browser window to various widths to confirm the footer adjusts appropriately for small, medium, and large screens.

# Where is it documented?
https://lavenderanyango.atlassian.net/browse/TG-85
